server/public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Favicon file is in https://github.com/xsolve-pl/xsolve-feat/blob/master/client/src/favicon.ico. You may try running https://github.com/xsolve-pl/xsolve-feat/blob/master/client/build-prod.sh to build SPA to `server/public` and check if it gets copied.